README.md

```diff
@@ -1,8 +1,13 @@
 # setup-lxd
 
-A GitHub Action which installs and configures the LXD snap on a runner.
+A GitHub Action installs and configures the LXD snap on a runner.
```

Comment:
This change doesn't make a lot of sense, perhaps a mistake?

Reply:
grammarly.com hint. Updated.